packages/fluentui/react-bindings/src/index.ts

```diff
@@ -11,25 +11,24 @@ export { default as FocusZone } from './FocusZone/FocusZone';
 export * from './FocusZone/FocusZone.types';
```

Comment:
There are still default exports in `react-bindings`:
![image](https://user-images.githubusercontent.com/9615899/86405855-5e44f700-bcb2-11ea-8d38-c3dc451865de.png)

Reply:
Yep, they will be handled separately as they require changes in `getComponentInfo()`: https://github.com/microsoft/fluentui/blob/d55eecf434dfe269ecbf6f412dcde73a13dace3e/scripts/gulp/plugins/util/getComponentInfo.ts#L64

All `export default` will be catched by ESLint rule https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-default-export.md. I will enable it in the final PR